deploy/local_single_user_deploy.sh

```diff
@@ -465,34 +389,38 @@ usage() {
     echo "  Note: Only one execution mode can be selected per run."
     echo ""
     echo "Build Options:"
-    echo "  -t|--build-type     Define type of build: Debug | Release (default: Release) [Modes: Build]"
-    echo "  -l|--install-dir    Define installation directory (default: /home/$USER/chronolog/BUILD_TYPE) [Modes: Build]"
+    echo "  -t|--build-type <Debug|Release>  Define type of build (default: Release) [Modes: Build]"
+    echo "  -l|--install-dir <path>          Define installation directory (default: /home/$USER/chronolog/BUILD_TYPE) [Modes: Build]"
     echo ""
     echo "Deployment Options:"
-    echo "  -k|--keepers        Set the total number of keeper processes. They will be assigned iteratively to the recording groups [Modes: Start]"
-    echo "  -r|--record-groups  Set the number of recording groups or grapher processes [Modes: Start]"
+    echo "  -k|--keepers <number>            Set the total number of keeper processes. They will be assigned iteratively to the recording groups (default: 1)[Modes: Start]"
+    echo "  -r|--record-groups <number>      Set the number of recording groups or grapher processes (default: 1)[Modes: Start]"
     echo ""
     echo "Directory Settings:"
-    echo "  -w|--work-dir       WORK_DIR Set the working directory (Mandatory) [Modes: Install, Start, Stop, Clean]"
-    echo "  -m|--monitor-dir    MONITOR_DIR (default: work_dir/monitor) [Modes: Start]"
-    echo "  -u|--output-dir     OUTPUT_DIR (default: work_dir/output) [Modes: Start]"
+    echo "  -w|--work-dir <path>             Set the working directory (Mandatory) [Modes: Install, Start, Stop, Clean]"
```

Comment:
can we merge `work_dir` with `install-dir`? they are the same directory where we install binaries, libs and confs to, and also where we copy and link all dependencies to.